backend/src/database/programs.go

```diff
@@ -12,6 +28,32 @@ func (db *DB) GetProgramByID(id int) (*models.Program, error) {
 	return content, nil
 }
 
+func (db *DB) FetchEnrollmentMetrics(programID int, facilityId uint) (ProgramOverviewResponse, error) {
```

Comment:
@corypride This function should return `EnrollmentAndCompletionMetrics` not `ProgramOverviewResponse`